### PR TITLE
manifest: remove .git suffixes from repository paths

### PR DESCRIPTION
**Pull request by gmarull**
.git suffix cannot be used to form valid URLs.

### CODE REVIEW

File: west.yml [hunk 1]
```diff
@@ -61,7 +61,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: 8d93a161567dcc525771f70692587e6a37947b71
+      revision: pull/1563/head
```
Comment (jfischer-no) [inline west.yml:64]: 7d195f1a20bdb1da597e92e6b88382c6e8aa0cf7
https://github.com/nrfconnect/sdk-zephyr/commit/7d195f1a20bdb1da597e92e6b88382c6e8aa0cf7

**Review by shanthanordic — APPROVED** [review]

**Review by umapraseeda — APPROVED** [review]

**Review by jfischer-no — APPROVED** [review]

File: west.yml [hunk 2]
```diff
@@ -61,7 +61,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: 8d93a161567dcc525771f70692587e6a37947b71
+      revision: 226b676f94f6d0999b3199a6723057f533754ac7
```
Comment (SeppoTakalo) [inline west.yml:64]: Is this supposed to be on its own commit?
Or why is it sneaked in?
Commit message does not describe the change.

Comment (gmarull) [reply west.yml:64]: well, manifest change pulls same kind of changes

**Review by SeppoTakalo — APPROVED** [review]
